modules/programs/fff.nix

```diff
@@ -0,0 +1,594 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+
+  cfg = config.programs.fff;
+
+in {
+  meta.maintainers = [ maintainers.gigahawk ];
+
+  options.programs.fff = {
+    enable = mkEnableOption "A simple file manager written in bash";
+
+    cdOnExit = {
+      enable = mkOption {
+        type = types.bool;
+        default = true;
+        description = ''
+          Change directory to current folder on exit
+        '';
+      };
+
+      name = mkOption {
```

Comment:
Maybe I'm missing something but I wasn't able to find any place where this option is used.